suite-common/wallet-utils/src/sendFormUtils.ts

```diff
@@ -185,10 +187,13 @@ export const prepareEthereumTransaction = (
         result.data = sanitizeHex(txInfo.data);
     }
 
-    // Build erc20 'transfer' method
+    // Build erc20 'transfer' method or use provided data in case of approve/revoke transaction
     if (txInfo.token) {
```

Comment:
the changes in this condition together with the condition itself is confusing. Why token is defined in case of approval/revoke?